src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/events/AttachDocsToSupplementaryEvidence.java

```diff
@@ -1,22 +1,37 @@
 package uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.events;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.stereotype.Component;
 import uk.gov.hmcts.reform.bulkscan.orchestrator.model.ccd.CaseData;
 import uk.gov.hmcts.reform.bulkscan.orchestrator.model.ccd.mappers.SupplementaryEvidenceMapper;
 import uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus.model.Envelope;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
 import uk.gov.hmcts.reform.ccd.client.model.StartEventResponse;
 
 import static uk.gov.hmcts.reform.bulkscan.orchestrator.helper.ScannedDocumentsHelper.getDocuments;
 
 @Component
 class AttachDocsToSupplementaryEvidence extends AbstractEventPublisher {
+    private static final Logger log = LoggerFactory.getLogger(AbstractEventPublisher.class);
 
     private final SupplementaryEvidenceMapper mapper;
 
     AttachDocsToSupplementaryEvidence(SupplementaryEvidenceMapper mapper) {
         this.mapper = mapper;
     }
 
+    public void handle(Envelope envelope, CaseDetails existingCase) {
+        log.info("Adding new evidence to case {} from envelope {}", existingCase.getId(), envelope.id);
```

Comment:
The log of start of event publishing is already present in abstract publisher. Do we want to add extra log here? Maybe we can take extra review on this and record this particular log when we actually begin adding new evidence.

The following if statement is more of an extra validation so we should be doing all of that and once the horizon is clear - record the beginning